src/simulation/m_riemann_solvers.fpp

```diff
@@ -336,6 +352,16 @@ contains
         real(wp) :: Ms_L, Ms_R, pres_SL, pres_SR
         real(wp) :: alpha_L_sum, alpha_R_sum
 
+        real(wp) :: Bx_L, Bx_R, By_L, By_R, Bz_L, Bz_R
+        real(wp) :: c_fast_L, c_fast_R
+
+        real(wp) :: pres_mag_L, pres_mag_R
+
+        real(wp) :: Ga_L, Ga_R ! Lorentz factor
+        real(wp) :: vdotB_L, vdotB_R, B2_L, B2_R
+        real(wp) :: b4x_L, b4y_L, b4z_L, b4x_R, b4y_R, b4z_R ! 4-magnetic field components
+        real(wp) :: mx_L, my_L, mz_L, mx_R, my_R, mz_R ! Conservative momentum variables
```

Comment:
can we change all of the `mx_L` type variables to `mx%L`? or perhaps even `m%x%L`?
alternatively m(1,1) for (x,L), m(2,2) for (y,R), etc. same for all of the `_L and `_R`s more intrusively. i know this is a lot but really makes the code simpler.